### PR TITLE
Create super

### PR DESCRIPTION
**Pull request by sachinpasi**
# Welcome to JavaScript community

[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new JavaScript files are placed inside an existing directory.
* [ ] All filenames should use the UpperCamelCase (PascalCase) style.  There should be no spaces in filenames.
     **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: super [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    }
+
+    get area() {
+        return this.height * this.width;
+    }
+
+    set area(value) {
+        this._area = value;
+    }
```
Comment (Khez) [inline super:20]: This doesn't make much sense...

File: super [hunk 2]
```diff
@@ -0,0 +1,29 @@
+class rectangle {
```
Comment (Khez) [inline super:1]: Why is `rectangle` lowercase while `Square` is capitalised ?

File: super [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    }
+}
+
+class Square extends rectangle {
+    constructor(length) {
+        this.height;
+        super(length, length);
+        this.name = 'Square';
+    }
+}
```
Comment (Khez) [inline super:29]: It would be helpful to have an example usage for people who want to understand how this works.

**Review by Khez — CHANGES_REQUESTED** [review]

**Review by ruppysuppy — COMMENTED** [review]
Go through the [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md) before adding any Pull Request

**Comment by ruppysuppy** [comment]
Irrelevant PR

**Pull request closed without merging**